pkg/asset/manifests/ingress.go

```diff
@@ -108,13 +108,24 @@ func (ing *Ingress) generateClusterConfig(config *types.InstallConfig) ([]byte,
 			Name: "cluster",
 			// not namespaced
 		},
-		Spec: configv1.IngressSpec{
-			Domain: fmt.Sprintf("apps.%s", config.ClusterDomain()),
-		},
+		Spec: configv1.IngressSpec{},
 		Status: configv1.IngressStatus{
 			DefaultPlacement: defaultPlacement,
 		},
 	}
+	obj.Spec.Domain = fmt.Sprintf("apps.%s", config.ClusterDomain())
+	if config.AWS != nil && len(config.AWS.LBType) != 0 {
+		obj.Spec = configv1.IngressSpec{
+			LoadBalancer: configv1.LoadBalancer{
+				Platform: configv1.IngressPlatformSpec{
+					AWS: &configv1.AWSIngressSpec{
+						Type: config.AWS.LBType,
+					},
+					Type: configv1.AWSPlatformType,
+				},
+			},
+		}
+	}
```

Comment:
when LBType is not provided in install-config, do we want to set it to "Classic" here?

Reply:
If no value is set then ingress operator will create a CR for Classic(same old behaviour) and then default Ingress Controller with Classic will get created. Setting LBType is not made mandatory so I have not set any default value.